processes/local_development/seed_test_data.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, *args):
             'uuid' : uuid4(),
             'date_created' : datetime.now(timezone.utc).replace(tzinfo=None),
             'date_modified' :datetime.now(timezone.utc).replace(tzinfo=None),
```

Comment:
You may not need to set these. I think these are set in the base postgres model class.